Parser/Client/Browser.php

```diff
@@ -570,6 +570,7 @@ class Browser extends AbstractClientParser
         'LE' => 'Smart Lenovo Browser',
         'OZ' => 'Smooz',
         'SN' => 'Snowshoe',
+        'K5' => 'Spark',
```

Comment:
Googling seems to suggest that there have been only "Baidu Spark" browser (which we do have already). Is this the one or is it something different?

Reply:
Is another browser, made by https://fastcorporate.com/ - https://www.advanceduninstaller.com/Spark-e1f42f2b16bcd71251a972fdf9b3d716-application.htm